documentation/staging/content/faq/external-clients.md

```diff
@@ -114,6 +120,8 @@ is hosted outside of the Kubernetes cluster, then:
 - You may need to [enable unknown host access](#enabling-unknown-host-access)
   on the WebLogic Servers that host the EJB or JMS resources.
 
+- You may need to configure a default load balancer algorithm that provides server affinity, for the cluster in which EJB or JMS resources are targted, to enable appropriate routing of requests via replica-aware stubs.
```

Comment:
1) You may -->  "If tunneling, "

2) Please add a link to the new section for changing the algorithm.   

3) "replica-aware stub" is an internal term IMO - so I recommend changing "
to enable appropriate routing of requests via replica-aware stubs" to 
something like "to speedup connection creation for EJB and JMS clients."